File: Pythonleetcode/Leetcode1108.py
class Solution(object):
    def defangIPaddr(self, address):
        ans=""
        for i in range(len(address)):
            if address[i] =='.':
                ans += '[.]'
            else:
                ans=ans + address[i]
        return ans






print(Solution().defangIPaddr("1.1.1.1"))

File: CppPractice/Project1/Project1/Leetcode1108.cpp
#include <string>

#include <vector>

#include <iostream>

#include <stdlib.h>

#include <time.h>

#include <sstream>

#include <algorithm>

#include <map>

using namespace std;

class Solution {
public:
    string defangIPaddr(string address) {
        string ans;
        for (int  i = 0; i < address.size(); i++)
        {
            if (address[i]!='.')
            {
                ans += address[i];

            }
            else 
            {
                ans +="[.]";
            
            }
        }

        return ans;



    }
};